CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Unreleased
 ------
 
+Version 1.7.1
+------
+* Add argo-admin-beta flag
```

Comment:
Does this affect external users? If so, we might want to make the message a little friendlier. Otherwise, we could just say something like `Updating internal features in development`, or some such.

Reply:
That makes sense!